smac/intensification/hyperband.py

```diff
@@ -116,43 +112,52 @@ def __init__(self,
         self.hb_iters = 0
         self.sh_intensifier = None  # type: SuccessiveHalving # type: ignore[assignment]
 
-    def eval_challenger(self,
+    def process_results(self,
                         challenger: Configuration,
                         incumbent: typing.Optional[Configuration],
                         run_history: RunHistory,
-                        time_bound: float = float(MAXINT),
-                        log_traj: bool = True) -> typing.Tuple[Configuration, float]:
+                        time_bound: float,
+                        result: RunValue,
+                        log_traj: bool = True,
+                        ) -> \
+            typing.Tuple[Configuration, float]:
         """
-        Running intensification via hyperband to determine the incumbent configuration.
-        *Side effect:* adds runs to run_history
-
-        Implementation of hyperband (Li et al., 2018)
+        The intensifier stage will be updated based on the results/status
+        of a configuration execution.
+        Also, a incumbent will be determined.
 
         Parameters
         ----------
         challenger : Configuration
-            promising configuration
-        incumbent : typing.Optional[Configuration]
-            best configuration so far, None in 1st run
-        run_history : smac.runhistory.runhistory.RunHistory
+            A configuration that was previously executed, and whose status
+            will be used to define the next stage.
+        incumbet : Configuration
```

Comment:
There's a typo here, should be incumbe**n**t

Reply:
Sorry, looked at the wrong commits.

Reply:
fixed!